inc/fulltext.php

```diff
@@ -279,10 +288,42 @@ function _ft_pageLookup(&$data){
         }
     }
 
+    $pages = _ft_filterResultsByTime($pages);
+
     uksort($pages,'ft_pagesorter');
     return $pages;
 }
 
+
+/**
+ * @param array $results search results in the form pageid => value
+ *
+ * @return array
+ */
+function _ft_filterResultsByTime(array $results) {
+    global $INPUT;
+    if ($INPUT->has('after') || $INPUT->has('before')) {
```

Comment:
It's a bit unfortunate that his function uses global input parameters. I'm not sure if we should/can fix this now or should wait til we refactor this file into a proper class.